packages/api/src/controllers/user.ts

```diff
@@ -1510,86 +1510,4 @@ app.post(
   },
 );
 
-const OLD_USER_CUTOFF = 1656312726000; // 27 June 2022
```

Comment:
Oh good practice. We should probably remove some other migration APIs that never got cleaned up :disappear: